pkg/sql/catalog/tabledesc/table_desc_test.go

```diff
@@ -150,6 +150,39 @@ func TestStripDanglingBackReferencesAndRoles(t *testing.T) {
 			strippedDanglingBackReferences: true,
 			strippedNonExistentRoles:       true,
 		},
+		{
+			name: "LDR job IDs",
+			input: descpb.TableDescriptor{
+				Name: "foo",
+				ID:   104,
+				MutationJobs: []descpb.TableDescriptor_MutationJob{
```

Comment:
this test case should not require mutation jobs, right?

Reply:
that's right; but just wanted to have a test that cleans up both types of jobs on the same descriptor